MonoGame.Framework/Input/MouseCursor.SDL.cs

```diff
@@ -34,21 +34,29 @@ private static void PlatformInitalize()
 
         private static MouseCursor PlatformFromTexture2D(Texture2D texture, int originx, int originy)
         {
-            IntPtr handle;
+            if (texture.Format != SurfaceFormat.Color && texture.Format != SurfaceFormat.ColorSRgb)
```

Comment:
We should probably make this a global rule and move it to: https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Input/MouseCursor.cs#L81